<!DOCTYPE html>
<html lang="en-us">

<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 4.5.0">

  

  
  
  
  
  
    
    
    
  
  

  <meta name="author" content="Dr. Peter Crisp">

  
  
  
    
  
  <meta name="description" content="In recent years, Setaria viridis has been developed as a model plant to better understand the C4 photosynthetic pathway in major crops. With the increasing availability of genomic resources for S. viridis research, highly efficient genome editing technologies are needed to create genetic variation resources for functional genomics. Here, we developed a protoplast assay to rapidly optimize the multiplexed CRISPR/Cas9 system in S. viridis. Targeted mutagenesis efficiency was further improved by an average of 1.4-fold with the exonuclease, Trex2. Distinctive mutation profiles were found in the Cas9_Trex2 samples with 94% of deletions larger than 10 bp, and essentially no insertions at all tested target sites. Further analyses indicated that 52.2% of deletions induced by Cas9_Trex2, as opposed to 3.5% by Cas9 alone, were repaired through microhomology-mediated end joining (MMEJ) rather than the canonical NHEJ DNA repair pathway. Combined with a robust Agrobacterium-mediated transformation method with more than 90% efficiency, the multiplex CRISPR/Cas9_Trex2 system was demonstrated to induce targeted mutations in two tightly linked genes, svDrm1a and svDrm1b, at the frequency ranging from 73% to 100% in T0 plants. These mutations were transmitted to at least 60% of the transgene-free T1 plants with 33% of them containing bi-allelic or homozygous mutations in both genes. This highly efficient multiplex CRISPR/Cas9_Trex2 system makes it possible to create a large mutant resource for S. viridis in a rapid and high throughput manner, and has the potential to be widely applicable in achieving more predictable and deletion-only MMEJ-mediated mutations in many plant species.">

  
  <link rel="alternate" hreflang="en-us" href="https://CrispLab.github.io/publication/weiss-optimization-2020/">

  


  
  
  
  <meta name="theme-color" content="#3f51b5">
  

  
  
  
  
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.0/css/all.css" integrity="sha384-aOkxzJ5uQz7WBObEZcHvV5JvRW3TUc2rNPA7pe3AwnsUohiw1Vj2Rgx2KSOkF5+h" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">

    
    
    
      
    
    
      
      
        
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/github.min.css" crossorigin="anonymous" title="hl-light">
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/dracula.min.css" crossorigin="anonymous" title="hl-dark" disabled>
        
      
    

    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.2.0/leaflet.css" integrity="sha512-M2wvCLH6DSRazYeZRIm1JnYyh22purTM+FDB5CsyxtQJYeKq83arPe5wgbNmcFXGqiSH2XR8dT/fJISVA1r/zQ==" crossorigin="anonymous">
    

    

  

  
  
  
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:400,700%7CRoboto:400,400italic,700%7CRoboto+Mono&display=swap">
  

  
  
  
  <link rel="stylesheet" href="/css/academic.min.b587b0a9a6a97d7e8dbc1fa62e1d4f23.css">

  

  




  


  

  <link rel="manifest" href="/index.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon-32.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="https://CrispLab.github.io/publication/weiss-optimization-2020/">

  
  
  
  
    
  
  
  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="twitter:site" content="@@pete_crisp">
  <meta property="twitter:creator" content="@@pete_crisp">
  
  <meta property="og:site_name" content="CrispLab">
  <meta property="og:url" content="https://CrispLab.github.io/publication/weiss-optimization-2020/">
  <meta property="og:title" content="Optimization of multiplexed CRISPR/Cas9 system for highly efficient genome editing in Setaria viridis | CrispLab">
  <meta property="og:description" content="In recent years, Setaria viridis has been developed as a model plant to better understand the C4 photosynthetic pathway in major crops. With the increasing availability of genomic resources for S. viridis research, highly efficient genome editing technologies are needed to create genetic variation resources for functional genomics. Here, we developed a protoplast assay to rapidly optimize the multiplexed CRISPR/Cas9 system in S. viridis. Targeted mutagenesis efficiency was further improved by an average of 1.4-fold with the exonuclease, Trex2. Distinctive mutation profiles were found in the Cas9_Trex2 samples with 94% of deletions larger than 10 bp, and essentially no insertions at all tested target sites. Further analyses indicated that 52.2% of deletions induced by Cas9_Trex2, as opposed to 3.5% by Cas9 alone, were repaired through microhomology-mediated end joining (MMEJ) rather than the canonical NHEJ DNA repair pathway. Combined with a robust Agrobacterium-mediated transformation method with more than 90% efficiency, the multiplex CRISPR/Cas9_Trex2 system was demonstrated to induce targeted mutations in two tightly linked genes, svDrm1a and svDrm1b, at the frequency ranging from 73% to 100% in T0 plants. These mutations were transmitted to at least 60% of the transgene-free T1 plants with 33% of them containing bi-allelic or homozygous mutations in both genes. This highly efficient multiplex CRISPR/Cas9_Trex2 system makes it possible to create a large mutant resource for S. viridis in a rapid and high throughput manner, and has the potential to be widely applicable in achieving more predictable and deletion-only MMEJ-mediated mutations in many plant species."><meta property="og:image" content="https://CrispLab.github.io/img/website%20icon.jpg">
  <meta property="twitter:image" content="https://CrispLab.github.io/img/website%20icon.jpg"><meta property="og:locale" content="en-us">
  
    
      <meta property="article:published_time" content="2020-09-16T08:10:54&#43;00:00">
    
    <meta property="article:modified_time" content="2020-01-01T00:00:00&#43;00:00">
  

  


    











<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "Article",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://CrispLab.github.io/publication/weiss-optimization-2020/"
  },
  "headline": "Optimization of multiplexed CRISPR/Cas9 system for highly efficient genome editing in Setaria viridis",
  
  "datePublished": "2020-09-16T08:10:54Z",
  "dateModified": "2020-01-01T00:00:00Z",
  
  "author": {
    "@type": "Person",
    "name": "Trevor Weiss"
  },
  
  "publisher": {
    "@type": "Organization",
    "name": "CrispLab",
    "logo": {
      "@type": "ImageObject",
      "url": "https://CrispLab.github.io/img/icon-512.png"
    }
  },
  "description": "In recent years, Setaria viridis has been developed as a model plant to better understand the C4 photosynthetic pathway in major crops. With the increasing availability of genomic resources for S. viridis research, highly efficient genome editing technologies are needed to create genetic variation resources for functional genomics. Here, we developed a protoplast assay to rapidly optimize the multiplexed CRISPR/Cas9 system in S. viridis. Targeted mutagenesis efficiency was further improved by an average of 1.4-fold with the exonuclease, Trex2. Distinctive mutation profiles were found in the Cas9_Trex2 samples with 94% of deletions larger than 10 bp, and essentially no insertions at all tested target sites. Further analyses indicated that 52.2% of deletions induced by Cas9_Trex2, as opposed to 3.5% by Cas9 alone, were repaired through microhomology-mediated end joining (MMEJ) rather than the canonical NHEJ DNA repair pathway. Combined with a robust Agrobacterium-mediated transformation method with more than 90% efficiency, the multiplex CRISPR/Cas9_Trex2 system was demonstrated to induce targeted mutations in two tightly linked genes, svDrm1a and svDrm1b, at the frequency ranging from 73% to 100% in T0 plants. These mutations were transmitted to at least 60% of the transgene-free T1 plants with 33% of them containing bi-allelic or homozygous mutations in both genes. This highly efficient multiplex CRISPR/Cas9_Trex2 system makes it possible to create a large mutant resource for S. viridis in a rapid and high throughput manner, and has the potential to be widely applicable in achieving more predictable and deletion-only MMEJ-mediated mutations in many plant species."
}
</script>

  

  


  


  





  <title>Optimization of multiplexed CRISPR/Cas9 system for highly efficient genome editing in Setaria viridis | CrispLab</title>

</head>

<body id="top" data-spy="scroll" data-offset="70" data-target="#TableOfContents" >

  <aside class="search-results" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search">
        
      </div>

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>


  
<nav class="navbar navbar-light fixed-top navbar-expand-lg py-0 compensate-for-scrollbar" id="navbar-main">
  <div class="container">

    
      <a class="navbar-brand" href="/">CrispLab</a>
      
      <button type="button" class="navbar-toggler" data-toggle="collapse"
              data-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
        <span><i class="fas fa-bars"></i></span>
      </button>
      

    
    <div class="collapse navbar-collapse" id="navbar">

      
      
      <ul class="navbar-nav ml-auto">
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#slider"><span>Home</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#posts"><span>News</span></a>
        </li>

        
        

        
        <li class="nav-item dropdown">
          <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown" aria-haspopup="true"><span>Research</span><span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="dropdown-item my-0 py-0 mx-0 px-0">
              <a href="/#projects"><span>Overview</span></a>
            </li>
            
            <li class="dropdown-item my-0 py-0 mx-0 px-0">
              <a href="/project/crop_epigenetics"><span>Crop Epigenetics</span></a>
            </li>
            
            <li class="dropdown-item my-0 py-0 mx-0 px-0">
              <a href="/project/umr-distilling-functional-elements"><span>Epigenomic Signatures</span></a>
            </li>
            
            <li class="dropdown-item my-0 py-0 mx-0 px-0">
              <a href="/project/stress"><span>Stress Resilience</span></a>
            </li>
            
          </ul>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#people"><span>People</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#featured"><span>Publications</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#contact"><span>Contact</span></a>
        </li>

        
        

      

        

        
        <li class="nav-item">
          <a class="nav-link js-search" href="#"><i class="fas fa-search" aria-hidden="true"></i></a>
        </li>
        

        

        
        <li class="nav-item">
          <a class="nav-link js-dark-toggle" href="#"><i class="fas fa-moon" aria-hidden="true"></i></a>
        </li>
        

      </ul>

    </div>
  </div>
</nav>


  <div class="pub">

  












  

  
  
  
<div class="article-container pt-3">
  <h1>Optimization of multiplexed CRISPR/Cas9 system for highly efficient genome editing in Setaria viridis</h1>

  

  
    



<div class="article-metadata">

  
  
  
  
  <div>
    



  
  <span><a href="/authors/trevor-weiss/">Trevor Weiss</a></span>, <span><a href="/authors/chunfang-wang/">Chunfang Wang</a></span>, <span><a href="/authors/xiaojun-kang/">Xiaojun Kang</a></span>, <span><a href="/authors/hui-zhao/">Hui Zhao</a></span>, <span><a href="/authors/maria-elena-gamo/">Maria Elena Gamo</a></span>, <span><a href="/authors/colby-g.-starker/">Colby G. Starker</a></span>, <span><a href="/authors/peter-a.-crisp/">Peter A. Crisp</a></span>, <span><a href="/authors/peng-zhou/">Peng Zhou</a></span>, <span><a href="/authors/nathan-m.-springer/">Nathan M. Springer</a></span>, <span><a href="/authors/daniel-f.-voytas/">Daniel F. Voytas</a></span>, <span><a href="/authors/feng-zhang/">Feng Zhang</a></span>

  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January 2020
  </span>
  

  

  

  
  
  

  
  

  
    
<div class="share-box" aria-hidden="true">
  <ul class="share">
    
      
      
      
        
      
      
      
      <li>
        <a href="https://twitter.com/intent/tweet?url=https://CrispLab.github.io/publication/weiss-optimization-2020/&amp;text=Optimization%20of%20multiplexed%20CRISPR/Cas9%20system%20for%20highly%20efficient%20genome%20editing%20in%20Setaria%20viridis" target="_blank" rel="noopener" class="share-btn-twitter">
          <i class="fab fa-twitter"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="https://www.facebook.com/sharer.php?u=https://CrispLab.github.io/publication/weiss-optimization-2020/&amp;t=Optimization%20of%20multiplexed%20CRISPR/Cas9%20system%20for%20highly%20efficient%20genome%20editing%20in%20Setaria%20viridis" target="_blank" rel="noopener" class="share-btn-facebook">
          <i class="fab fa-facebook-f"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="mailto:?subject=Optimization%20of%20multiplexed%20CRISPR/Cas9%20system%20for%20highly%20efficient%20genome%20editing%20in%20Setaria%20viridis&amp;body=https://CrispLab.github.io/publication/weiss-optimization-2020/" target="_blank" rel="noopener" class="share-btn-email">
          <i class="fas fa-envelope"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="https://www.linkedin.com/shareArticle?url=https://CrispLab.github.io/publication/weiss-optimization-2020/&amp;title=Optimization%20of%20multiplexed%20CRISPR/Cas9%20system%20for%20highly%20efficient%20genome%20editing%20in%20Setaria%20viridis" target="_blank" rel="noopener" class="share-btn-linkedin">
          <i class="fab fa-linkedin-in"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="https://web.whatsapp.com/send?text=Optimization%20of%20multiplexed%20CRISPR/Cas9%20system%20for%20highly%20efficient%20genome%20editing%20in%20Setaria%20viridis%20https://CrispLab.github.io/publication/weiss-optimization-2020/" target="_blank" rel="noopener" class="share-btn-whatsapp">
          <i class="fab fa-whatsapp"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="https://service.weibo.com/share/share.php?url=https://CrispLab.github.io/publication/weiss-optimization-2020/&amp;title=Optimization%20of%20multiplexed%20CRISPR/Cas9%20system%20for%20highly%20efficient%20genome%20editing%20in%20Setaria%20viridis" target="_blank" rel="noopener" class="share-btn-weibo">
          <i class="fab fa-weibo"></i>
        </a>
      </li>
    
  </ul>
</div>


  

</div>

    













<div class="btn-links mb-3">
  
  








  
    
  



<a class="btn btn-outline-primary my-1 mr-1" href="https://onlinelibrary.wiley.com/doi/abs/10.1111/tpj.14949" target="_blank" rel="noopener">
  PDF
</a>















<a class="btn btn-outline-primary my-1 mr-1" href="https://doi.org/10.1111/tpj.14949" target="_blank" rel="noopener">
  DOI
</a>



</div>


  
</div>



  <div class="article-container">

    
    <h3>Abstract</h3>
    <p class="pub-abstract">In recent years, Setaria viridis has been developed as a model plant to better understand the C4 photosynthetic pathway in major crops. With the increasing availability of genomic resources for S. viridis research, highly efficient genome editing technologies are needed to create genetic variation resources for functional genomics. Here, we developed a protoplast assay to rapidly optimize the multiplexed CRISPR/Cas9 system in S. viridis. Targeted mutagenesis efficiency was further improved by an average of 1.4-fold with the exonuclease, Trex2. Distinctive mutation profiles were found in the Cas9_Trex2 samples with 94% of deletions larger than 10 bp, and essentially no insertions at all tested target sites. Further analyses indicated that 52.2% of deletions induced by Cas9_Trex2, as opposed to 3.5% by Cas9 alone, were repaired through microhomology-mediated end joining (MMEJ) rather than the canonical NHEJ DNA repair pathway. Combined with a robust Agrobacterium-mediated transformation method with more than 90% efficiency, the multiplex CRISPR/Cas9_Trex2 system was demonstrated to induce targeted mutations in two tightly linked genes, svDrm1a and svDrm1b, at the frequency ranging from 73% to 100% in T0 plants. These mutations were transmitted to at least 60% of the transgene-free T1 plants with 33% of them containing bi-allelic or homozygous mutations in both genes. This highly efficient multiplex CRISPR/Cas9_Trex2 system makes it possible to create a large mutant resource for S. viridis in a rapid and high throughput manner, and has the potential to be widely applicable in achieving more predictable and deletion-only MMEJ-mediated mutations in many plant species.</p>
    

    
    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Type</div>
          <div class="col-12 col-md-9">
            
            
            <a href="/publication/#2">
              Journal article
            </a>
            
          </div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>
    

    
    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Publication</div>
          <div class="col-12 col-md-9"><em>The Plant Journal</em></div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>
    

    <div class="space-below"></div>

    <div class="article-style"></div>

    

<div class="article-tags">
  
  <a class="badge badge-light" href="/tags/crispr/cas9/">CRISPR/Cas9</a>
  
  <a class="badge badge-light" href="/tags/multiplexed-genome-editing/">Multiplexed genome editing</a>
  
  <a class="badge badge-light" href="/tags/nhej-and-mmej-dna-repair/">NHEJ and MMEJ DNA repair</a>
  
  <a class="badge badge-light" href="/tags/setaria-viridis/">Setaria viridis</a>
  
  <a class="badge badge-light" href="/tags/trex2-exonuclease/">Trex2 exonuclease</a>
  
</div>


    








  
  
    
  
  






  
  
  
  
  <div class="media author-card">
    

    <div class="media-body">
      <h5 class="card-title"><a href="/authors/trevor-weiss/"></a></h5>
      
      
      <ul class="network-icon" aria-hidden="true">
  
</ul>

    </div>
  </div>




    


  </div>
</div>



      

    
    

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js" integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js" integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>

      

      
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/highlight.min.js" integrity="sha256-aYTdUrn6Ow1DDgh5JTc3aDGnnju48y/1c8s1dgkYPQ8=" crossorigin="anonymous"></script>
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/languages/r.min.js"></script>
        
      

      
      
    

    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.2.0/leaflet.js" integrity="sha512-lInM/apFSqyy1o6s89K4iQUKg6ppXEgsVxT35HbzUupEVRh2Eu9Wdl4tHj7dZO0s1uvplcYGmt3498TtHq+log==" crossorigin="anonymous"></script>
    

    
    
    <script>hljs.initHighlightingOnLoad();</script>
    

    
    
    
    
    
    
    <script>
      const search_config = {"indexURI":"/index.json","minLength":1,"threshold":0.3};
      const i18n = {"no_results":"No results found","placeholder":"Search...","results":"results found"};
      const content_type = {
        'post': "Posts",
        'project': "Projects",
        'publication' : "Publications",
        'talk' : "Talks"
        };
    </script>
    

    
    

    
    
    <script id="search-hit-fuse-template" type="text/x-template">
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js" integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script>
    

    
    

    
    

    
    
    
    
    
    
    
    
    
      
    
    
    
    
    <script src="/js/academic.min.d6bd04fdad2ad213aa8111c5a3b72fc5.js"></script>

    






  
  <div class="container">
    <footer class="site-footer">
  

  <p class="powered-by">
    CC by-nd-nc 4.0; 2019 &middot; 

    Powered by the
    <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic theme</a> for
    <a href="https://gohugo.io" target="_blank" rel="noopener">Hugo</a>.

    
    <span class="float-right" aria-hidden="true">
      <a href="#" class="back-to-top">
        <span class="button_icon">
          <i class="fas fa-chevron-up fa-2x"></i>
        </span>
      </a>
    </span>
    
  </p>
</footer>

  </div>
  

  
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

</body>
</html>
